app/scripts/modules/core/pipeline/config/stages/bake/bakeStageChooseOs.component.html

```diff
@@ -0,0 +1,9 @@
+<ui-select ng-model="model" required class="form-control input-sm" on-select="$ctrl.updateOnSelect($item)">
```

Comment:
If you change this to `ng-model="$ctrl.model"`, I think you can drop `constructor`, `$onInit`, and `$inject` from the controller. (And not using `$scope` is more in line with how we're writing new components).